src/session.ts

```diff
@@ -1,13 +1,13 @@
-import { redirect } from 'next/navigation';
+import { sealData, unsealData } from 'iron-session';
+import { createRemoteJWKSet, decodeJwt, jwtVerify } from 'jose';
 import { cookies, headers } from 'next/headers';
+import { redirect } from 'next/navigation';
 import { NextRequest, NextResponse } from 'next/server';
-import { jwtVerify, createRemoteJWKSet, decodeJwt } from 'jose';
-import { sealData, unsealData } from 'iron-session';
 import { cookieName, cookieOptions } from './cookie.js';
-import { workos } from './workos.js';
 import { WORKOS_CLIENT_ID, WORKOS_COOKIE_PASSWORD, WORKOS_REDIRECT_URI } from './env-variables.js';
 import { getAuthorizationUrl } from './get-authorization-url.js';
 import { AccessToken, AuthkitMiddlewareAuth, NoUserInfo, Session, UserInfo } from './interfaces.js';
+import { workos } from './workos.js';
```

Comment:
Why the re-org of imports?

Reply:
Just what the `prettier` doctor order it seems. Can revert 👍